sdk/internal/recording/matchers.go

```diff
@@ -0,0 +1,42 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package recording
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+)
+
+// Optional parameters for the AddBodilessMatcher operation
+type MatcherOptions struct {
+}
+
+// AddBodilessMatcher adjusts the "match" operation to exclude the body when matching a request to a recording's entries.
+// Pass in `nil` for `t` if you want the bodiless matcher to apply everywhere
+func AddBodilessMatcher(t *testing.T, options *MatcherOptions) error {
```

Comment:
What do you think of `SetBodilessMatcher`? The rationale being that "set" is more correct semantically because the proxy only uses one matcher at a time; this method doesn't add to a collection but instead replaces the current matcher